sunpy/visualization/animator/wcs.py

```diff
@@ -223,7 +225,6 @@ def update_plot(self, val, artist, slider):
             self.update_plot_2d(val, artist, slider)
 
         self._apply_coord_params(self.axes)
-        return super().update_plot(val, artist, slider)
```

Comment:
Why the delete?

Reply:
I think it always returns `None`, and is never used.

Reply:
The base class has logic in it's update_plot method: https://github.com/sunpy/sunpy/blob/master/sunpy/visualization/animator/base.py#L614 so unless I am missing something, I think this should stay.